### PR TITLE
python310Packages.sanic: 22.3.2 -> 22.6.2

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 189268` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages marked as broken and skipped:
</summary>
<ul>
<li>python310Packages.pytest-sanic</li>
<li>python39Packages.pytest-sanic</li>
</ul>
</details>
<details>

  <summary>17 packages failed to build and are new build failures:</summary>
  <ul>
<li>dyndnsc: log was empty</li>
<li>python310Packages.elastic-apm: log was empty</li>
<li>python310Packages.entrance: log was empty</li>
<li>python310Packages.entrance-with-router-features: log was empty</li>
<li>python310Packages.json-logging: log was empty</li>
<li>python310Packages.sanic: <a href="https://termbin.com/y0mga">plain log</a></li>
<li>python310Packages.sanic-auth: log was empty</li>
<li>python310Packages.sanic-testing: log was empty</li>
<li>python310Packages.strawberry-graphql: log was empty</li>
<li>python39Packages.elastic-apm: log was empty</li>
<li>python39Packages.entrance: log was empty</li>
<li>python39Packages.entrance-with-router-features: log was empty</li>
<li>python39Packages.json-logging: log was empty</li>
<li>python39Packages.sanic: <a href="https://termbin.com/noom">plain log</a></li>
<li>python39Packages.sanic-auth: log was empty</li>
<li>python39Packages.sanic-testing: log was empty</li>
<li>python39Packages.strawberry-graphql: log was empty</li>
  </ul>
</details>
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python310Packages.aioquic</li>
<li>python310Packages.pylsqpack</li>
<li>python39Packages.aioquic</li>
<li>python39Packages.pylsqpack</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python310Packages.entrance:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python310Packages.entrance-with-router-features:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python310Packages.sanic:
<details>


warning: unused-argument
Unused argument: `pytest-benchmark`.
Near pkgs/development/python-modules/sanic/default.nix:12:3:

```
   |
12 | , pytest-benchmark
   |   ^
```
warning: unused-argument
Unused argument: `pytest-sugar`.
Near pkgs/development/python-modules/sanic/default.nix:13:3:

```
   |
13 | , pytest-sugar
   |   ^
```
warning: unused-argument
Unused argument: `pythonAtLeast`.
Near pkgs/development/python-modules/sanic/default.nix:16:3:

```
   |
16 | , pythonAtLeast
   |   ^
```
</details>
python310Packages.sanic-testing:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/sanic-testing/default.nix:50:

```
   |
50 |     description = "Core testing clients for the Sanic web framework";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.entrance:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.entrance-with-router-features:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.sanic:
<details>


warning: unused-argument
Unused argument: `pytest-benchmark`.
Near pkgs/development/python-modules/sanic/default.nix:12:3:

```
   |
12 | , pytest-benchmark
   |   ^
```
warning: unused-argument
Unused argument: `pytest-sugar`.
Near pkgs/development/python-modules/sanic/default.nix:13:3:

```
   |
13 | , pytest-sugar
   |   ^
```
warning: unused-argument
Unused argument: `pythonAtLeast`.
Near pkgs/development/python-modules/sanic/default.nix:16:3:

```
   |
16 | , pythonAtLeast
   |   ^
```
</details>
python39Packages.sanic-testing:
<details>


warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/sanic-testing/default.nix:50:

```
   |
50 |     description = "Core testing clients for the Sanic web framework";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 189268` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages marked as broken and skipped:
</summary>
<ul>
<li>python310Packages.pytest-sanic</li>
<li>python39Packages.pytest-sanic</li>
</ul>
</details>
<details>
<summary>
21 packages built:
</summary>
<ul>
<li>dyndnsc</li>
<li>python310Packages.aioquic</li>
<li>python310Packages.elastic-apm</li>
<li>python310Packages.entrance</li>
<li>python310Packages.entrance-with-router-features</li>
<li>python310Packages.json-logging</li>
<li>python310Packages.pylsqpack</li>
<li>python310Packages.sanic</li>
<li>python310Packages.sanic-auth</li>
<li>python310Packages.sanic-testing</li>
<li>python310Packages.strawberry-graphql</li>
<li>python39Packages.aioquic</li>
<li>python39Packages.elastic-apm</li>
<li>python39Packages.entrance</li>
<li>python39Packages.entrance-with-router-features</li>
<li>python39Packages.json-logging</li>
<li>python39Packages.pylsqpack</li>
<li>python39Packages.sanic</li>
<li>python39Packages.sanic-auth</li>
<li>python39Packages.sanic-testing</li>
<li>python39Packages.strawberry-graphql</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

<details>
python310Packages.entrance:
<details>

got build log for '/nix/store/m3322zd6qwlymqg77gqyy3fpvyc8wx6b-python3.10-entrance-1.1.17' from 'daemon'
When evaluating attribute ‘python310Packages.entrance’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python310Packages.entrance-with-router-features:
<details>

got build log for '/nix/store/nphjb8j3s7vny961bil80q4s2rasflfr-python3.10-entrance-1.1.17' from 'daemon'
When evaluating attribute ‘python310Packages.entrance-with-router-features’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python310Packages.sanic:
<details>

got build log for '/nix/store/mpw04xa8msaiyflbqsarmwjdx7rd9ns2-python3.10-sanic-22.6.2' from 'daemon'
When evaluating attribute ‘python310Packages.sanic’:
warning: unused-argument
Unused argument: `pytest-benchmark`.
Near pkgs/development/python-modules/sanic/default.nix:12:3:

```
   |
12 | , pytest-benchmark
   |   ^
```
warning: unused-argument
Unused argument: `pytest-sugar`.
Near pkgs/development/python-modules/sanic/default.nix:13:3:

```
   |
13 | , pytest-sugar
   |   ^
```
warning: unused-argument
Unused argument: `pythonAtLeast`.
Near pkgs/development/python-modules/sanic/default.nix:16:3:

```
   |
16 | , pythonAtLeast
   |   ^
```
</details>
python310Packages.sanic-testing:
<details>

got build log for '/nix/store/91xxj4lscgmv82wcykgjszzn6z018by8-python3.10-sanic-testing-22.3.1' from 'daemon'
When evaluating attribute ‘python310Packages.sanic-testing’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/sanic-testing/default.nix:50:

```
   |
50 |     description = "Core testing clients for the Sanic web framework";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.entrance:
<details>

got build log for '/nix/store/sln8brin3cqwja6j3q19p48jxmilvf92-python3.9-entrance-1.1.17' from 'daemon'
When evaluating attribute ‘python39Packages.entrance’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.entrance-with-router-features:
<details>

got build log for '/nix/store/1slr7cm9hdbwiq719w62xzg7j55gs44g-python3.9-entrance-1.1.17' from 'daemon'
When evaluating attribute ‘python39Packages.entrance-with-router-features’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/entrance/default.nix:39:

```
   |
39 |     description = "A server framework for web apps with an Elm frontend";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
python39Packages.sanic:
<details>

got build log for '/nix/store/n3fv6np1js8la7a7jkvwccshpjldrzaz-python3.9-sanic-22.6.2' from 'daemon'
When evaluating attribute ‘python39Packages.sanic’:
warning: unused-argument
Unused argument: `pytest-benchmark`.
Near pkgs/development/python-modules/sanic/default.nix:12:3:

```
   |
12 | , pytest-benchmark
   |   ^
```
warning: unused-argument
Unused argument: `pytest-sugar`.
Near pkgs/development/python-modules/sanic/default.nix:13:3:

```
   |
13 | , pytest-sugar
   |   ^
```
warning: unused-argument
Unused argument: `pythonAtLeast`.
Near pkgs/development/python-modules/sanic/default.nix:16:3:

```
   |
16 | , pythonAtLeast
   |   ^
```
</details>
python39Packages.sanic-testing:
<details>

got build log for '/nix/store/p0799pxc3a83b38b6p1gy025kwp93aam-python3.9-sanic-testing-22.3.1' from 'daemon'
When evaluating attribute ‘python39Packages.sanic-testing’:
warning: python-include-tests
Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/sanic-testing/default.nix:50:

```
   |
50 |     description = "Core testing clients for the Sanic web framework";
   | ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md
</details>
</details>

File: pkgs/development/python-modules/sanic/default.nix [hunk 1]
```diff
@@ -20,11 +20,12 @@
 , uvicorn
 , uvloop
 , websockets
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sanic/default.nix:22]: Please cleanup the inputs.

**Comment by dotlambda** [comment]
https://github.com/sanic-org/sanic/security/advisories/GHSA-8cw9-5hmv-77w6

**Review by risicle — APPROVED** [review]
`nixpkgs-review` happy, aarch64-linux

**Comment by risicle** [comment]
`nixpkgs-review` also reveals no new failures, macos 10.15 & nixos x86_64

**Pull request merged**